lib/rubyai.rb

```diff
@@ -31,17 +40,16 @@ def call
 
     def body
       {
-        'model': model,
-        'messages': [{"role": "user", "content": messages}],
-        'temperature': temperature
+        'model': MODELS[@model],
```

Comment:
- Do we have a default model now? 
- Can we add def mode:l gpt-4
- Can you add some unit test for test model versions

Reply:
Default model is gpt 3.5 turbo, but ill add instance to choose model set by default